client/dashboard/me/billing-purchases/cancel-purchase/info-popover/index.jsx

```diff
@@ -0,0 +1,171 @@
+import { Popover, Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import { createRef, Component, Fragment } from 'react';
+// import { gaRecordEvent } from 'calypso/lib/analytics/ga';
+
+import './style.scss';
+
+export default class InfoPopover extends Component {
```

Comment:
I haven't compared the two but you can probably use `BillingPurchaseInfoPopover` that I implemented for the purchases list instead of making a new one.